config/zones/US-CAL-CISO.yaml

```diff
@@ -11,17 +11,23 @@ capacity:
     - datetime: '2023-12-01'
       source: EIA.gov
       value: 7985.9
+    - datetime: '2024-01-01'
+      source: EIA.gov
+      value: 8056.5
     - datetime: '2024-03-01'
       source: EIA.gov
       value: 8430.5
+    - datetime: '2024-04-01'
+      source: EIA.gov
+      value: 8836.1
   biomass:
     - datetime: '2023-10-01'
       source: EIA.gov
       value: 993.1
     - datetime: '2023-12-01'
       source: EIA.gov
       value: 970.9
-    - datetime: '2024-03-01'
```

Comment:
Is this intentional?

Reply:
Yes, the script deduplicates the datetimes so only the first recorded datetime with that capacity is kept. When I re-ran it I ran it month by month for this year this updated some of the datetimes as well.